lib/synapse_pay_rest/http_client.rb

```diff
@@ -123,8 +123,10 @@ def get(path)
     # @raise [SynapsePayRest::Error] subclass depends on HTTP response
     # 
     # @return [Hash] API response
```

Comment:
It probably makes sense to add the new `payload` arg to the doc comment. I'm mostly thinking in the context of potentially getting this merged into the main repo.